sql/src/main/java/org/opensearch/sql/sql/parser/AstExpressionBuilder.java

```diff
@@ -9,6 +9,7 @@
 import static org.opensearch.sql.ast.dsl.AstDSL.between;
 import static org.opensearch.sql.ast.dsl.AstDSL.not;
 import static org.opensearch.sql.ast.dsl.AstDSL.qualifiedName;
+import static org.opensearch.sql.expression.function.BuiltinFunctionName.DATETIME;
```

Comment:
typo or incomplete fix?

Reply:
typo, removed in https://github.com/Bit-Quill/opensearch-project-sql/pull/297/commits/966efbb082b78fca450cd436641d3a3fe7a81842